clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -557,7 +608,7 @@ void ConstraintBuilderConsumer::HandleTranslationUnit(ASTContext &C) {
   }
 
 
-  VariableAdderVisitor VAV = VariableAdderVisitor(&C, Info);
+  VariableAdderVisitor VAV = VariableAdderVisitor(&C, Info, Info);
```

Comment:
This is confusing. Why are you using `Info` twice? Maybe talk to @jackastner about the right way to do this. He's the one that abstracted out `ProgramInfo` into being an instance of the `VariableAddr` class, IIRC.